mods/tuxemon/maps/spyder_paper_town.tmx

```diff
@@ -423,13 +423,14 @@
     <property name="act61" value="npc_face player,down"/>
     <property name="act70" value="translated_dialog spyder_papertown_firstfight"/>
     <property name="act80" value="start_battle spyder_billie"/>
-    <property name="act81" value="translated_dialog spyder_papertown_firstfight2"/>
+    <property name="act81" value="translated_dialog spyder_papertown_firstfight_win"/>
+    <property name="cond1" value="is player_defeated"/>
```

Comment:
This condition is currently incorrect - it should return `true` if the NPC was defeated, not the player.